nodes/parachain/src/chain_spec/politest.rs

```diff
@@ -192,7 +192,7 @@ pub fn get_live_chain_spec() -> GenericChainSpec {
 
 #[allow(clippy::too_many_arguments)]
 fn testnet_genesis(
-	stakers: Vec<(AccountId, Option<AccountId>, Balance)>,
+	#[allow(unused)] stakers: Vec<(AccountId, Option<AccountId>, Balance)>,
```

Comment:
Same for this?

Reply:
because compiling with runtime-benchmarks gives a warning